supportedBackends/google/batch/src/main/scala/cromwell/backend/google/batch/runnable/RunnableUtils.scala

```diff
@@ -16,30 +16,16 @@ object RunnableUtils {
   /** Port mappings for the ssh container. */
   val sshPortMappings = Map("22" -> Int.box(22))
 
-  /*
-   * At the moment, cloud-sdk (924MB for 276.0.0-slim) and stedolan/jq (182MB) decompressed ~= 1.1 GB
-   */
-  val cromwellImagesSizeRoundedUpInGB = 1
```

Comment:
It looks like we're removing all the image size stuff from `RunnableUtils`, is that right?

Reply:
The variable is unused. I bugged Dan Speck just now to confirm that is deliberate.

Reply:
[Confirmation received](https://chat.google.com/room/AAAAVRKNtz4/4K_7vgHeFOY/IvCZ5LCFiEI?cls=10) (requires auth)